frontend/src/individual_certificate.tsx

```diff
@@ -370,26 +420,46 @@ export function IndividualCertificate({ id }: { id: string }) {
               color="primary"
               disabled={isUserGood ? false : true}
               onClick={() => {
-                issueCertificate(id, certificateData);
+                switch (action) {
+                  case "issue":
+                    issueCertificate(id, certificateData);
+                    break;
+                  case "update":
+                    updateCertificate(id, certificateData);
+                }
               }}
             >
-              {"Beställ certifikat"}
+              {action == "issue"
+                ? "Beställ certifikat"
+                : "update"
+                ? "Uppdatera certifikat"
+                : "Fortsätt"}
             </Button>
           </div>
         </>
       ) : individual && showComplete ? (
         <>
-          <h1>Certifikatet är klart!</h1>
-          <a target="_blank" href={certificateUrl} download={individual.number}>
-            <Button variant="contained" color="primary">
-              {"Ladda ner"}
-            </Button>
-          </a>
-          <a target="_blank" href={certificateUrl}>
-            <Button variant="contained" color="primary">
-              {"Öppna i ny flik"}
-            </Button>
-          </a>
+          {action == "issue" ? (
+            <h1>Certifikatet är klart!</h1>
+          ) : (
+            <h1>Certifikatet uppdaterades!</h1>
+          )}
+          <div className={style.paneControls}>
+            <a
+              target="_blank"
+              href={certificateUrl}
+              download={individual.number}
```

Comment:
Do we really need `target="_blank"` here? I have not yet succeeded in testing, but my understanding is that the `download` tag should trigger a file download in anything resembling modern (I'm yet unable to understand if `download` and `target="_blank"` will give a flashing window that disappear, an empty window or something else).

Reply:
@HenrikeW Will this be addressed here or in the other PR?